README.md

```diff
@@ -1 +1,34 @@
-# capstone-project
+This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
```

Comment:
I think this readMe should be updated with info on your project at some point

Reply:
Thanks, i have updated my readme :)